scripts/cache_cleaner.py

```diff
@@ -0,0 +1,32 @@
+import json
```

Comment:
Where it is planned to use this script?

Reply:
In case of corrupted cache as it was today on master branch -- you can delete cache manually via this script